### ISSUE TITLE
Focus changes on form entry.

### ISSUE THREAD

**Issue by mbstacy** [culibraries/ir-scholar#61]
After activating a limiting option (e.g. Masters Thesis) the page reloads and screen reader focus is pushed to the top of the page.  It would be beneficial if screen reader focus was instead moved to the top of the results section of the page, as that would limit the additional navigation required to determine if the limit was applied and to get to the actual results.

The same thing occurs when removing a constraint (screen reader focus is pushed to the top of the page).

Additionally, when using NVDA, the tester had to press the Tab key to allow for any navigation on the reloaded page.  If the user did not press the Tab key first, they were unable to use arrow key navigation on the page at all after the reload.  This seems to be the case any time the page reloads after an action.
